packages/docs/src/components/BasicNiivueCanvas.jsx

```diff
@@ -1,6 +1,15 @@
-import React from "react";
+import React, { useRef, useEffect } from "react";
 import { Niivue } from "@niivue/niivue";
+import BrowserOnly from "@docusaurus/BrowserOnly";
 
+<<<<<<< HEAD
+export const NiivueApp = ({ images, nvOpts }) => {
+  const canvasRef = useRef(null);
+  const niivue = useRef(null);
+
+  useEffect(() => {
+    niivue.current = new Niivue({ logLevel: "debug", ...nvOpts });
+=======
 export const BasicNiivueCanvas = ({ images, nvOpts }) => {
```

Comment:
We want to keep this name BasicNiivueCanvas, not NiivueApp. This component is meant to be a single example for others to use as a starting point. New components should have their own files and unique names.